core/types/liquidity.go

```diff
@@ -412,7 +413,7 @@ func (l LiquidityMonitoringParameters) IntoProto() *proto.LiquidityMonitoringPar
 	}
 	return &proto.LiquidityMonitoringParameters{
 		TargetStakeParameters: params,
-		TriggeringRatio:       tr,
+		TriggeringRatio:       strconv.FormatFloat(tr, 'f', -1, 64),
```

Comment:
This should just be `l.TriggeringRatio.String()`, currently we're going from `Decimal -> float64 -> string`

Reply:
Corrected here as well https://github.com/vegaprotocol/vega/pull/7222/commits/935f9774a523f334d55e43742ff63338043eb2c4#diff-8bf56342620e6cec70bbb347efe846b5a22a316a7e903809ebe5707a7ffe0386R414
Thanks!